cpp/include/gar/graph.h

```diff
@@ -205,6 +205,13 @@ class VertexIter {
     return ret;
   }
 
+  /** The add operator. */
```

Comment:
You should add a in-place add operator like ++ in line 196 to support in-place add